packages/backend-modules/publikator/graphql/resolvers/_mutations/publish.js

```diff
@@ -139,7 +139,9 @@ module.exports = async (_, args, context) => {
     utm_campaign: repoId,
   }
 
-  const searchString = '?' + querystring.stringify(utmParams)
+  const base64Email = 'email=*|EMAILB64U|*'
+
+  const searchString = '?' + querystring.stringify(utmParams) + '&' + base64Email
```

Comment:
@lucianak shouldn't this just be `querystring.stringify({...utmParams, email: "*|EMAILB64U|*"})`?

Reply:
or even better:

```
const params = {
    utm_source: 'newsletter',
    utm_medium: 'email',
    utm_campaign: repoId,
    email: '*|EMAILB64U|*'
  }

const searchString = '?' + querystring.stringify(params)
  ```